### DISCUSSION TITLE
SharePoint挂载后访问提示PROPFIND /: 403

### DISCUSSION THREAD

**Discussion by kettly1260** [AlistGo/alist#1389] [Q&A]
国内教育版onedrive，没有管理员只能用webdav挂载SharePoint。
但是填入相关配置后访问后提示403错误，但相同配置可以在rclone正常使用，请问这个是怎么回事啊 
![image](https://user-images.githubusercontent.com/20242614/180677421-8944d367-189e-48f3-9c56-c9e7ea72e483.png)

**Comment by SkysCrystal** [comment]
同问

**Comment by yangdunyu** [comment]
同问，alist和raidrive我试了楼主说的方法，都连不上(403)，楼主说rclone可以，我看论坛也有人说rclone可以，还没试
